### PR TITLE
[OCRVS 6971] Image compression reduction configuration

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
Countryconfig PR: https://github.com/opencrvs/opencrvs-countryconfig/pull/132

### CODE REVIEW

File: packages/client/src/components/form/DocumentUploadfield/DocumentUploaderWithOption.tsx [hunk 1]
```diff
@@ -201,26 +193,26 @@ class DocumentUploaderWithOptionComp extends React.Component<
...
-        this.props.onUploadingStateChanged(false)
-
-      this.setState({
-        errorMessage:
-          this.state.errorMessage ||
-          this.props.intl.formatMessage(messages.uploadError),
-        // Remove from processing files
-        filesBeingProcessed: this.state.filesBeingProcessed.filter(
+      if (props.onUploadingStateChanged) {
+        props.onUploadingStateChanged(true)
```
Comment (Zangetsu101) [inline packages/client/src/components/form/DocumentUploadfield/DocumentUploaderWithOption.tsx:201]:
```suggestion
        props.onUploadingStateChanged(false)
```

File: packages/client/src/components/form/DocumentUploadfield/DocumentUploaderWithOption.tsx [hunk 2]
```diff
@@ -234,124 +226,110 @@ class DocumentUploaderWithOptionComp extends React.Component<
...
-        },
-        dropDownOptions: tempOptions,
-        // Remove from processing files
-        filesBeingProcessed: this.state.filesBeingProcessed.filter(
-          ({ label }) => label !== optionValues[1]
-        )
-      }
+    props.onComplete([...props.files, newDocument])
+    if (props.onUploadingStateChanged) {
+      props.onUploadingStateChanged(true)
```
Comment (Zangetsu101) [inline packages/client/src/components/form/DocumentUploadfield/DocumentUploaderWithOption.tsx:231]:
```suggestion
      props.onUploadingStateChanged(false)
```

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

**Review by Zangetsu101 — APPROVED** [review]

**Comment by Zangetsu101** [comment]
@tahmidrahman-dsi Could you please add a changelog for this PR and briefly explain there what the 2 new field properties enable for us?

**Comment by tahmidrahman-dsi** [comment]
> @tahmidrahman-dsi Could you please add a changelog for this PR and briefly explain there what the 2 new field properties enable for us?

@Zangetsu101 done https://github.com/opencrvs/opencrvs-core/pull/7019/commits/c9ff82f8e4f55bf3b6448d5667f537aa1219c69b

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**